wmn-data.json

```diff
@@ -1201,6 +1201,22 @@
         "known" : ["jaxjames", "jordan-2"],
         "cat" : "xx NSFW xx"
        },
+       {
+        "name": "coub",

```

Comment:
Please change the `name` to `Coub` since that is how it is capitalized on the site.